src/package.xml

```diff
@@ -364,6 +372,7 @@
         <members>Facility__c.Account__c</members>
         <members>Facility__c.Capacity__c</members>
         <members>Facility__c.Description__c</members>
+        <members>Facility__c.Display_Name__c</members>
```

Comment:
@kdy618 I think you mistakenly added this to the package.xml. We deploy this as unmanaged, so it shouldn't be included in the package.

Reply:
@robertbwatson, it's all the Facility__c related fields that should be removed from the package.xml right?

Reply:
@kdy618 Sorry for not being clear - it looks like you added the Display_Name__c field in one of your recent commits. We don't actually want that field in our package.